gypbuild.js

```diff
@@ -32,6 +32,12 @@ function runGyp (opts, target, cb) {
 
     if (opts.format) args.push('--', '--format', opts.format)
 
+    for (var key of Object.keys(opts)) {
+      if (key.substr(0, 2) === 'BK') {
+        args.push('--' + key.substr(2) + '=' + opts[key])
+      }
+    }
```

Comment:
What happens here if `opts.format` is set or not set? 

Compare `cmd foo bar -- --format someting --msvs_version=2013`
versus `cmd foo bar --msvs_version=2013`

Reply:
Also this code is in two places and identical